gordo_components/cli/cli.py

```diff
@@ -62,6 +67,8 @@
 logger = logging.getLogger(__name__)
 logging.getLogger("azure.datalake").setLevel(azure_log_level)
 
+cv_mode_input = {"cross_val_only", "full_build", "build_only"}
```

Comment:
So we'll have to update both the documentation for available choices, _and_ remember to update this choice list, which lives in a completely separate module?